src/qibo/result.py

```diff
@@ -26,10 +26,15 @@ class QuantumState:
     """Data structure to represent the final state after circuit execution.
 
     Args:
-
+        state (np.ndarray): Input quantum state as np.ndarray.
+        backend (qibo.backends.AbstractBackend): Backend used for the calculations. If not provided the :class:`qibo.backends.GlobalBackend` is going to be used.
     """
 
-    def __init__(self, state, backend):
+    def __init__(self, state, backend=None):
+        if backend is None:
+            from qibo.backends import GlobalBackend
```

Comment:
This could go to the top if it doesn't create a circular import error

Reply:
it does